RecoLocalTracker/SiPhase2Clusterizer/plugins/BuildFile.xml

```diff
@@ -0,0 +1,4 @@
+<use name="RecoLocalTracker/SiPhase2Clusterizer"/>
+<library   file="Phase2TrackerClusterizer.cc ../src/Phase2TrackerClusterizerAlgorithm.cc ../src/Phase2TrackerClusterizerArray.cc" name="Phase2TrackerClusterizerPlugin">
```

Comment:
either move the code from src/ to plugins/ or let the parts live  in separate libraries.